magellan-library/src/main/java/com/wealthfront/magellan/Screen.java

```diff
@@ -46,6 +49,8 @@
   private boolean dialogIsShowing;
   private Dialog dialog;
   private SparseArray<Parcelable> viewState;
+  private boolean transitionFinished = false;
```

Comment:
We could also default this to true, allowing view updates to happen before the transition starts